packages/components/src/components/link-tile/link-tile-styles.ts

```diff
@@ -63,16 +70,10 @@ export const getComponentCss = (
   hasGradient: boolean
 ): string => {
   const isTopAligned = align === 'top';
-  const paddingSizeXS = pxToRemWithUnit(24);
-  const gradientPadding = pxToRemWithUnit(72);
   return getCss({
     '@global': {
       ':host': {
-        display: 'block',
-        ...addImportantToEachRule({
-          height: 'fit-content',
-          ...hostHiddenStyles,
-        }),
+        ...addImportantToEachRule(hostHiddenStyles),
```

Comment:
I think we should keep `display: 'block'` or otherwise it behaves like an linline element?

Reply:
Done